src/routes/Settings/SettingsIndexView.js

```diff
@@ -1,7 +1,185 @@
 // @flow
 
-function SettingsIndexView() {
-  return null
+import React, { PureComponent } from 'react'
+
+import { JText } from 'components/base'
+import { SettingsGrid, SettingsCard } from 'components'
+
+import { numbersUtils } from 'utils'
+
+type Props = {|
+  ...SettingsState,
+  +networkName: string,
+  +walletType: WalletType,
+  +walletName: string,
+  +derivationPath: string,
+  +passphrase: string,
+|}
+
+class SettingsIndexView extends PureComponent<Props, *> {
+  formatBoolean = (value: boolean): string => value ? 'Enabled' : 'Disabled'
+
+  formatLanguage = (langCode: string): string => {
+    const languageMap = {
+      en: 'English',
+      kr: 'Korean',
+    }
+
+    return languageMap[langCode] || 'Language not found'
+  }
+
+  formatCurrency = (curCode: string): string => {
+    const currencyCode = {
+      usd: 'US Dollar',
+      rub: 'Russian Ruble',
+    }
+
+    return currencyCode[curCode.toLowerCase()] || 'Currency not found'
+  }
+
+  formatGasPrice = (price: string): string => numbersUtils.divideThousands(price)
```

Comment:
Looks like there is no need to define those as class methods.

You should use plain old functions instead.

After you get rid of methods, you can also move from PureComponent to functional component.

Reply:
I'm agree. It's pretty reasonable.